src/testPane.ts

```diff
@@ -314,12 +305,11 @@ function getEnvironmentList(baseDirectory: string): string[] {
 }
 
 // These variables are used in the package.json to control the display of context menu items
-// Search for 'vectorcastTestExplorer.vcastEnviroList | vcastHasCodedTestsList' in package.json 
+// Search for 'vectorcastTestExplorer.vcastEnviroList | vcastHasCodedTestsList' in package.json
 // to see where we reference them
 // this list in a "when" clause
-export var vcastEnviroList: string[] = [];
-export var vcastHasCodedTestsList: string[] = [];
-
+let vcastEnviroList: string[] = [];
```

Comment:
no need for export anymore (we're not using these in other files)?

Reply:
The export was removed in this change :)